src/Features/CSharp/CodeFixes/Async/CSharpAddAwaitCodeFixProvider.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Formatting;
 using Resources = Microsoft.CodeAnalysis.CSharp.CSharpFeaturesResources;
+using Roslyn.Utilities;
 
```

Comment:
Thanks, I've fixed this.  As an aside, though, this reference was added by Roslyn's auto-fix... should it be changed to respect a similar ordering, or do we consider this ordering specific to Roslyn's codebase and not something to be enforced as part of the auto-fix?